htdocs/modules/publisher/sql/schema.yml

```diff
@@ -874,8 +874,8 @@ tables:
                 columnDefinition: null
                 comment: null
         indexes:
-            mime_id:
-                name: mime_id
+            publisher_mimetypes_mime_id:
```

Comment:
Experienced an error when updating an existing installation with this one. On investigation, I can see no reason that particular index should not be declared as PRIMARY. That would also solve the unique name issue in this case. That change is made in #349 which includes all these changes, too.